pkg/consensus/commit.go

```diff
@@ -0,0 +1,23 @@
+package consensus
+
+import "github.com/CityOfZion/neo-go/pkg/io"
+
+type commit struct {
```

Comment:
at least will be nice to know why we use BE instead of LE here(while encoding/decoding)

Reply:
All message format are specified in C# implementation.

Reply:
ok got it. But it will be nice to have them here so it's not needed to go back and forth while reading the code